### PR TITLE
Volvo pr Ciel day 14 part 2

### PR DESCRIPTION
**Pull request by Volvo-hackangel**

### CODE REVIEW

File: script/Ciel/Day 14/14_01B_01_CIEL14B_1D.txt [hunk 1]
```diff
@@ -741,7 +741,7 @@ She has to dedicate half of her senses to regulating its still-rising internal p
 [sha:79a52e4f02dae429f4a34b78a977b382e2af11f2]{
 -- Page 188, Offset 38876.
 -- 流動する魔力による聖典詠唱の反復制御にまた半分を。
-The other half are busy controlling the flow of magical energy that drives the repeated scriptural chanting.
+The other half is busy controlling the flow of magical energy that drives the repeated scriptural chanting.
```
Comment (rschlaikjer) [inline script/Ciel/Day 14/14_01B_01_CIEL14B_1D.txt:744]: Senses are plural, are correct.

File: script/Ciel/Day 14/14_01B_01_CIEL14B_1D.txt [hunk 2]
```diff
@@ -951,12 +951,12 @@ This too had been part of Ciel's predictions.
...
 -- Page 200, Offset 38914.
 -- まず頭を潰して意識を刈り取る。
-First, she would take out her head and render her unconscious.
+First, Ciel would take out Arcueid's head and render her unconscious.
 }
 [sha:e0f9af5d600cf3a612a3ac4b196b71422fe2573f]{
 -- Page 200, Offset 38915.
 -- その後、彼女が思考能力を取り戻す前に本体を破壊する。
-Then, before Arcueid regains the ability to think clearly, she would obliterate her body.
+Then, before Arcueid regains the ability to think clearly, Ciel would obliterate her body.
```
Comment (rschlaikjer) [inline script/Ciel/Day 14/14_01B_01_CIEL14B_1D.txt:959]: I think keep her -> Arcueid in the first sentence, but leave the `she`s. Should be clear from context of prev page that Ciel is doing the actions.

File: script/Ciel/Day 14/14_01B_01_CIEL14B_1F.txt [hunk 3]
```diff
@@ -318,7 +318,7 @@ All of them―――
 -- Page 263, Offset 39128.
 -- 今の姫君に腕を振るわせるとは、称賛にあまりある”
 %{i}
-She deserves no faint praise for managing to display such strength when up against the current Princess."
+She deserves no faint praise for managing to display such strength when up against the current princess."
```
Comment (rschlaikjer) [inline script/Ciel/Day 14/14_01B_01_CIEL14B_1F.txt:321]: Capitalizing Princess may be a Roa-ism, check with Petrikow

File: script/Ciel/Day 14/14_01B_01_CIEL14B_1G.txt [hunk 4]
```diff
@@ -146,6 +146,7 @@ What makes it so terrifying is that it's a natural phenomenon,
 [offset:39180]{
 -- Page 281, Offset 39180.
 -- 「こ、の……！」
+// can this be a swear?
```
Comment (rschlaikjer) [inline script/Ciel/Day 14/14_01B_01_CIEL14B_1G.txt:149]: Ciel doesn't swear

File: script/Ciel/Day 14/14_01B_01_CIEL14B_1G.txt [hunk 5]
```diff
@@ -162,7 +163,7 @@ Ciel is constantly estimating how close she can get without receiving a fatal bl
 [sha:60c00e2eb0b1ecea6a61d61fec2d7d2a4ab5ad4c]{
 -- Page 282, Offset 39183.
 -- 対して、アルクェイドは何の工夫も注意もない。
-Arcueid, on the other hand, has no plan and no need for caution.
+Arcueid, on the other hand, has neither plan nor need for caution.
```
Comment (rschlaikjer) [inline script/Ciel/Day 14/14_01B_01_CIEL14B_1G.txt:166]: revert

File: script/Ciel/Day 14/14_01B_01_CIEL14B_1G.txt [hunk 6]
```diff
@@ -904,7 +905,7 @@ But one would do well to stand in awe of the majesty of the planet.
 -- Page 328, Offset 39313.
 -- 人智における最高峰の大魔術を受け、なお月の姫は倒れない。
 // First 月の姫??
-For even the pinnacle of grand magecraft devised by man cannot yet fell the princess of the moon.
+For even the pinnacle of grand magecraft devised by man cannot yet fell the Princess of the Moon.
```
Comment (rschlaikjer) [inline script/Ciel/Day 14/14_01B_01_CIEL14B_1G.txt:908]: Not sure if this should be caps, again defer to Petrikow

File: script/Ciel/Day 14/14_01B_02_CIEL14B_2.txt [hunk 7]
```diff
@@ -232,7 +232,7 @@ And right now, something similar,
 [sha:7e12b1b5fa1f9e2244eb9350e8d4468be00e0e9e]{
 -- Page 14, Offset 39386.
 -- 太陽が落ちたようだ。
-It's as if the sun had fallen to earth.
+It's as if the sun had fallen to the earth.
```
Comment (rschlaikjer) [inline script/Ciel/Day 14/14_01B_02_CIEL14B_2.txt:235]: I think original is fine?

File: script/Ciel/Day 14/14_01B_02_CIEL14B_2B.txt [hunk 8]
```diff
@@ -719,7 +719,7 @@ But there's no way in hell I'm going to just stand here and watch!
 -- Page 63, Offset 39554.
 -- 『あの状態の姫君に近づくのか。
 %{i}
-"So you intend to approach the Princess despite her current state?
+"So you intend to approach the princess despite her current state?
```
Comment (rschlaikjer) [inline script/Ciel/Day 14/14_01B_02_CIEL14B_2B.txt:722]: possible Roa artifact again, see prev comment

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3.txt [hunk 9]
```diff
@@ -569,7 +569,7 @@ That would %{i}definitely%{/i} get me noticed."
 -- Page 120, Offset 39757.
 -- 『無論、正面は避け、迂回するべきだ。
 %{i}
-"You ought naturally avoid going straight,
+"You ought to naturally avoid going straight,
```
Comment (rschlaikjer) [inline script/Ciel/Day 14/14_01B_02_CIEL14B_3.txt:572]: Revert, speech style

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3B.txt [hunk 10]
```diff
@@ -663,7 +663,7 @@ By losing herself in her own emotions,
 // MBTL (器の臨界が近いのか……？　百年の恋も冷める というものだ).
 // I feel like it makes the most sense that he's talking about himself.
 %{i}
-Even a millennia of love may fade at such a sight."
+Even a millennium of love may fade at such a sight."
```
Comment (rschlaikjer) [inline script/Ciel/Day 14/14_01B_02_CIEL14B_3B.txt:666]: I'd leave it non-specific
> Even millenia of love [...]

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3B.txt [hunk 11]
```diff
@@ -268,7 +268,7 @@ Just as I'm starting to feel confident that I'll make it there in one piece―
 -- Page 146, Offset 39832. Mods: @k, @e.
 -- 姫君は指先の獲物に夢中で、
 %{i}
-The Princess is so obsessed with the prey in her fingertips
+The princess is so obsessed with the prey in her fingertips
```
Comment (rschlaikjer) [inline script/Ciel/Day 14/14_01B_02_CIEL14B_3B.txt:271]: see prev

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3B.txt [hunk 12]
```diff
@@ -693,7 +693,7 @@ I am disappointed in Elesia.
 -- 大魔術だけで十分だったものを、必要以上に壊すとは。
 %{i}
 Even though her grand magecraft would have sufficed,
- she destroyed the Princess far more than was necessary.
+ she destroyed the princess far more than was necessary.
```
Comment (rschlaikjer) [inline script/Ciel/Day 14/14_01B_02_CIEL14B_3B.txt:696]: see prev

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3B.txt [hunk 13]
```diff
@@ -933,7 +933,8 @@ She totally noticed me ages ago and is just ignoring me, I mean, come on!
 [sha:738c22eeeb60e940e2545c5791d98bc25e7250c1]{
 -- Page 184, Offset 39941.
 -- 目測距離、10メートル。
-Distance: ten meters.
+// maybe keep this as digits to be consistent with 39953?
```
Comment (rschlaikjer) [inline script/Ciel/Day 14/14_01B_02_CIEL14B_3B.txt:936]: Sure

File: script/Ciel/Day 14/14_01B_01_CIEL14B_1B.txt [hunk 14]
```diff
@@ -177,14 +177,13 @@ My legs had been poised to run away,
 -- 片腕。
 Up comes an arm.
 }
-[sha:a557dba75ef13db1705bc451d0f1a1c9bb902d3b]{
+[offset:38474]{
```
[the remark this answers is not in the document]
Comment (Volvo-hackangel) [reply script/Ciel/Day 14/14_01B_01_CIEL14B_1B.txt:180]: I thought 37670 is exactly the same as this one.

File: script/Ciel/Day 14/14_01B_01_CIEL14B_1B.txt [hunk 15]
```diff
@@ -394,7 +393,7 @@ The man in my head extols this inscrutable phenomenon.%{n}
 %{i}
 This is the power of a True Ancestor.
  As the planet's sense of touch,
- her Marble Phantasm allows her to rewrite the planet's surface however she sees fit.
+ her Marble Phantasm allows her to rewrite the planet's surface how ever she sees fit.
```
[the remark this answers is not in the document]
Comment (Volvo-hackangel) [reply script/Ciel/Day 14/14_01B_01_CIEL14B_1B.txt:396]: I thought how ever and however are used differently.

File: script/Ciel/Day 14/14_01B_01_CIEL14B_1F.txt (hunk 3 above)
Comment (Volvo-hackangel) [reply script/Ciel/Day 14/14_01B_01_CIEL14B_1F.txt:321]: Is that so? I dont think we were consistent with it in Arc route. Might as well ask him though. @Petrikow 
image

File: script/Ciel/Day 14/14_01B_01_CIEL14B_1G.txt (hunk 4 above)
Comment (Volvo-hackangel) [reply script/Ciel/Day 14/14_01B_01_CIEL14B_1G.txt:149]: Should we change this one then?
https://github.com/Tsukihimates/Tsukihime-Translation/blob/3434928ca29651ebf7576ce28b7c96f572df10ad/script/Ciel/Day%2014/14_01B_02_CIEL14B_2.txt#L78

File: script/Ciel/Day 14/14_01B_01_CIEL14B_1G.txt (hunk 6 above)
Comment (Volvo-hackangel) [reply script/Ciel/Day 14/14_01B_01_CIEL14B_1G.txt:908]: I thought it's a title like Princess of the True Ancestors. A little help then? @Petrikow

File: script/Ciel/Day 14/14_01B_02_CIEL14B_2.txt (hunk 7 above)
Comment (Volvo-hackangel) [reply script/Ciel/Day 14/14_01B_02_CIEL14B_2.txt:235]: The new line sounded better in my head at first now not so much.

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3.txt (hunk 9 above)
Comment (Volvo-hackangel) [reply script/Ciel/Day 14/14_01B_02_CIEL14B_3.txt:572]: Yeah, this is like the third time. I should have known it was.

File: script/Ciel/Day 14/14_01B_01_CIEL14B_1G.txt (hunk 4 above)
Comment (rschlaikjer) [reply script/Ciel/Day 14/14_01B_01_CIEL14B_1G.txt:149]: That's the once exception

File: script/Ciel/Day 14/14_01B_01_CIEL14B_1G.txt (hunk 6 above)
Comment (Petrikow) [reply script/Ciel/Day 14/14_01B_01_CIEL14B_1G.txt:907]: It's not something that shows up particularly often.
Think it's used by Roa in MBAA and MBTL at points.
I guess it does sort of sound like a title, so why not capitalize it

File: script/Ciel/Day 14/14_01B_02_CIEL14B_2B.txt [hunk 16]
```diff
@@ -585,7 +585,7 @@ The 'soul' that ordinarily resides within the planet,
 -- 長く天体を運営する為の“魂”を地表に漏らす、
 %{force_glue}
 %{i}
- and is responsible for long term operation of the celestial body,
+ and is responsible for the long-term operation of the celestial body,
 %{/i}
 }
```
Comment (Petrikow) [inline script/Ciel/Day 14/14_01B_02_CIEL14B_2B.txt:590]: Uncountable "operation" is fine and normal

File: script/Ciel/Day 14/14_01B_01_CIEL14B_1D.txt [hunk 17]
```diff
@@ -421,15 +421,15 @@ Ordinarily, the True Ancestor's <terms of service|d r e s s  c o d e
 [sha:d373e9e94e4e23bf27db244fa11384d44602c612]{
 -- Page 171, Offset 38818.
 -- 真祖は星の分身であるため、その資源を人間のように自己利益に使う事ができない。その発想がない。
-Because True Ancestors are alter-egos of the planet, they cannot use those resources for their own benefit as humans do.
+Because True Ancestors are alter egos of the planet, they cannot use those resources for their own benefit as humans do.
  The thought simply doesn't occur to them.
```
[the remark this answers is not in the document]
Comment (Volvo-hackangel) [reply script/Ciel/Day 14/14_01B_01_CIEL14B_1D.txt:425]: Ive look around I dont think Ive seen it hyphenated.
image
image

File: script/Ciel/Day 14/14_01B_01_CIEL14B_1G.txt (hunk 6 above)
Comment (Volvo-hackangel) [reply script/Ciel/Day 14/14_01B_01_CIEL14B_1G.txt:907]: caps it is then

**Pull request merged**